csrankings-v.csv

```diff
@@ -228,7 +230,7 @@ Vijay Ganesh,University of Waterloo,https://ece.uwaterloo.ca/~vganesh,YP23eR0AAA
 Vijay K. Sood,Ontario Tech University,https://engineering.ontariotechu.ca/people/ecse/vijay.sood.php,3kp505oAAAAJ
 Vijay Kumar 0001,University of Pennsylvania,http://www.kumarrobotics.org,h6h5FdoAAAAJ
 Vijay Kumar 0002,University of Missouri - Kansas City,http://k.web.umkc.edu/kumarv/index.html,NOSCHOLARPAGE
-Vijay Nagarajan,University of Utah,http://homepages.inf.ed.ac.uk/vnagaraj,xQYIFX0AAAAJ
+Vijay Nagarajan,University of Edinburgh,http://homepages.inf.ed.ac.uk/vnagaraj,xQYIFX0AAAAJ

```

Comment:
Web page says "Professor (on a leave of absence)". Faculty on non-sabbatical leave are not included in the database; please remove.

Reply:
Sorry for this and thank you for your help! Vijay moved after I submitted the original pull request in November. Now fixed.